src/endpoint/s3/ops/s3_get_bucket.js

```diff
@@ -88,9 +88,11 @@ function cont_tok_to_key_marker(cont_tok) {
     }
 }
 
-function key_marker_to_cont_tok(key_marker) {
-    if (!key_marker) return;
-    const j = JSON.stringify({ key: key_marker });
+function key_marker_to_cont_tok(key_marker, objects_arr, is_truncated) {
```

Comment:
No consideration into `Common Prefixes`, they should be handled as well.

Reply:
Common prefixes are returned when the delimiter is provided. if a delimiter provided so we will have the next marker and won't need to create it by our self.

https://docs.aws.amazon.com/AmazonS3/latest/API/API_ListObjectsV2.html